windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -104,6 +115,8 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 [id="wmco-2-0-0-known-issues"]
 == Known issues
 
+* When you create Windows pods with `RunAsUserName` set in its "SecurityContext" with a projected volume associated with these pods, the file ownership permissions for the projected entities are ignored, preventing the Windows pods from operating.
```

Comment:
It does not prevent the Windows Pods from operating. It just does not configure the file ownership permissions correctly. The Pod will go to `Running`.